browser-test/src/support/admin_program_image.ts

```diff
@@ -16,10 +16,8 @@ export class AdminProgramImage {
     await waitForPageJsLoad(this.page)
   }
 
-  async setImageFileAndSubmit() {
-    await this.page
-      .locator('input[type=file]')
-      .setInputFiles('src/assets/program-summary-image.png')
+  async setImageFileAndSubmit(imageFileName: string) {
```

Comment:
praise:  I appreciate how tidy and succinct your tests are, probably thanks to this helper function.  Thanks for doing that!